google-chrome-launcher

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+chrome_flatpak_dir=`flatpak-spawn --host flatpak info -l com.google.Chrome`
+chrome_binary=${chrome_flatpak_dir}/files/extra/opt/google/chrome/google-chrome
+
+exec flatpak-spawn --host --env=LD_LIBRARY_PATH=${chrome_flatpak_dir}/files/lib ${chrome_binary} $@
```

Comment:
I think we should do more than this - actually fetch the library path(s) from the runtime, and use LD_INTERP - so that the linker and all of the libraries from the runtime are picked up.

Reply:
FWIW I was able to make this work, and I'm now hitting the sandbox issue with the `--user` installation.